src/ServiceControl.Transports.RabbitMQ/RabbitMQQuery.cs

```diff
@@ -93,6 +94,8 @@ protected override void InitializeCore(ReadOnlyDictionary<string, string> settin
             // so for now we are using a virtual method that can be overriden in tests
             // https://github.com/Particular/ServiceControl/issues/4493
             httpClient = CreateHttpClient(defaultCredential, apiUrl);
+            var authToken = Convert.ToBase64String(System.Text.Encoding.ASCII.GetBytes($"{username}:{password}"));
+            httpClient.DefaultRequestHeaders.Authorization = new AuthenticationHeaderValue("Basic", authToken);
```

Comment:
This sends the credentials in plain text which is not what we want and we would want to change this.

Using `PreAuthenticate = true,` on the httpclient instead of setting this header directly does not work.